microsoft-edge/webview2/release-notes.md

```diff
@@ -147,7 +147,180 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
 ###### Runtime-only
 ###### SDK-only
 -->
-<!-- end of Jan 2024 Prerelease SDK -->
+<!-- end of Mar. 2024 Prerelease SDK -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2415-prerelease
+
+Release Date: February 26, 2024
+
+[NuGet package for WebView2 SDK 1.0.2415-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2415-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 122.0.2415.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Breaking changes
+
+* Change in `LaunchingExternalURISchemeEventArgs` behavior.  If the `InitiatingOrigin` is an [opaque origin](https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-opaque),the `InitiatingOrigin` reported in the event args will be its precursor origin.<!-- todo: link for more info -->
+
+* The members of the `CoreWebView2TextureStreamErrorKind` enum have been renamed:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+Old member names:
+* [CoreWebView2TextureStreamErrorKind.NoVideoTrackStarted Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamerrorkind?view=webview2-dotnet-1.0.2357-prerelease&preserve-view=true)

```

Comment:
```suggestion
* [CoreWebView2TextureStreamErrorKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2texturestreamerrorkind?view=webview2-dotnet-1.0.2357-prerelease&preserve-view=true)

```

Reply:
already fixed.  (GH not responding to Resolve button rn)